common/config/rush/pnpm-lock.yaml

```diff
@@ -49,7 +49,7 @@ packages:
       is-buffer: 2.0.4
       jssha: 2.3.1
       process: 0.11.10
-      rhea: 1.0.19
+      rhea: 1.0.20
```

Comment:
How did removing dependency on mocha-multi result in rhea version getting updated?
Is this a result of running more than just `rush update`?

Reply:
No, this is due to `rush update --full`.

Reply:
If a library is removed, `pnpm-lock` file is unaffected with `rush update`.

Reply:
@KarishmaGhiya

Reply:
I thought the guidance was not to run `rush update --full` as part of regular PRs and that this would be done in frequent intervals by the Eng Sys team. This keeps the changes in the rest of the PR and the changes to lock file in sync

Reply:
cc @bterlson

Reply:
Okay, makes sense to remove the unrelated changes. 
I have reverted `pnpm-lock` file changes now. I can revert my reversion if required.

Reply:
https://github.com/Azure/azure-sdk-for-js/issues/6025#issuecomment-549600517

Perhaps, we should add the above somewhere in our contributing guide

Reply:
> I have reverted pnpm-lock file changes now

I dont see a commit for this and I still see unrelated changes in the lock file

Reply:
pushed!

Reply:
Is the guidance that we should have a separate PR that *only* does update --full then? Otherwise we won't be able to shake removed deps out.

Reply:
If we need to remove a dep ASAP, then yes I would recommend a second PR that only does "update --full".  Anyone on the JS team can create such a PR, or you can ask @KarishmaGhiya to do it on your behalf.

Alternatively, @KarishmaGhiya should be running "update --full" about once per week, and in most cases it should be fine to wait until this weekly update to remove unused deps.

Reply:
Of course, if a PR needs "update --full" to be green, then the PR should include an "update --full" commit in the same PR.

Reply:
The reason is that `update --full` updates **all** packages to semver latest, which often breaks something (and will often be unrelated to any other changes in your PR).  So it's safest to isolate `update --full` into a separate PR and ensure we run `core` tests on it.